tools/prost-build/.cargo/config

```diff
@@ -0,0 +1,5 @@
+# [source.crates-io]
+# replace-with = "crates-ioo"
+
+# [sources.crates-ioo]
+# registry = "https://github.com/rust-lang/crates.io-index"
```

Comment:
It's all commended anyway. I've just tried building after `rm -rf .cargo` and it still ok, no complaints from cargo. Remove?

Reply:
Or did you maybe mean to reuse the vendored deps from `src/rust/vendor`?

Reply:
Yeah was a debug leftover. I am not sure if we should vendor it, what do you think?

Reply:
I wouldn't vendor it just for the prost-build's sake but it's already vendored in `src/rust/vendor`, right. Could we use that?

Reply:
It would be a separate vendor folder, as the tool is a x86_64 cli build dep, and is totally independent of our firmware.

Reply:
I don't think it matters for vendored deps? It's always the same set of source files. When you build, then it picks which files to compile, right.

Reply:
Either way, I don't think it's that imporant to vendor deps for this little tool. I wouldn't do it.

Reply:
I am not sure to be honest. Gut feeling is to keep separate things separate, for clarity.

/tools and /src/rust are also separate folders, so using one shared vendor folder is also confusing.

Reply:
In any case, we can probably go without vendoring for now here, especially since the generated file is committed.

Reply:
:+1: